### PR TITLE
cleaning go.mod from github.com/go-logr/zapr

### LINKED ISSUE #331: Logger Enhancements
Because k8gb is constantly expanding, the requirements for monitoring the application are also increasing.
I'd like to mention a few of them:

 - DBG: There is no need to log everything all the time. Some messages only interests us when we debug the system or check its behavior. Cleaning log from too detailed information makes monitoring easier. On the other hand, when debugging system we need as detailed info as possible. I'd like to introduce boolean environment variable called `LOG_DEBUGGING`. When true, then prints `ERR + DBG + INFO + WRN`, otherwise `ERR + INFO + WRN`. 

- PROD / NONPROD : would be nice to distinguish between PROD and NONPROD systems. It could be Date format (for prod we want to show DateTime but only Time for nonprod). It could be format of message (plain print for NONPROD, json for PROD). Should distinguish by environment variable. For now bool variable `LOG_JSON`

- ERRORS handling: instead of printing entire stack trace including GO libraries, we should trace only stack related to k8gb

- please add your requirements for logging if any ?

To fulfill this purpose, I chose [zerolog](https://github.com/rs/zerolog) who has everything I mentioned earlier. In addition it has very high performance:

Library | Time | Bytes Allocated | Objects Allocated
-- | -- | -- | --
zerolog | 767 ns/op | 552 B/op | 6 allocs/op
⚡ zap | 848 ns/op | 704 B/op | 2 allocs/op
⚡ zap (sugared) | 1363 ns/op | 1610 B/op | 20 allocs/op
go-kit | 3614 ns/op | 2895 B/op | 66 allocs/op
lion | 5392 ns/op | 5807 B/op | 63 allocs/op
logrus | 5661 ns/op | 6092 B/op | 78 allocs/op
apex/log | 15332 ns/op | 3832 B/op | 65 allocs/op
log15 | 20657 ns/op | 5632 B/op | 93 allocs/op

Log a static string, without any context or printf-style templating:
Library | Time | Bytes Allocated | Objects Allocated
-- | -- | -- | --
zerolog | 50 ns/op | 0 B/op | 0 allocs/op
⚡ zap | 236 ns/op | 0 B/op | 0 allocs/op
standard library | 453 ns/op | 80 B/op | 2 allocs/op
⚡ zap (sugared) | 337 ns/op | 80 B/op | 2 allocs/op
go-kit | 508 ns/op | 656 B/op | 13 allocs/op
lion | 771 ns/op | 1224 B/op | 10 allocs/op
logrus | 1244 ns/op | 1505 B/op | 27 allocs/op
apex/log | 2751 ns/op | 584 B/op | 11 allocs/op
log15 | 5181 ns/op | 1592 B/op | 26 allocs/op

### PR DESCRIPTION
**Pull request by kuritka**
related to #331

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -318,9 +316,8 @@ github.com/go-logr/logr v0.1.0/go.mod h1:ixOQHD9gLJUVQQ2ZOR7zLEifBX6tGkNJF4QyIY7
 github.com/go-logr/logr v0.2.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
 github.com/go-logr/logr v0.4.0 h1:K7/B1jt6fIBQVd4Owv2MqGQClcgf0R266+7C/QjRcLc=
 github.com/go-logr/logr v0.4.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
+github.com/go-logr/zapr v0.1.0 h1:h+WVe9j6HAA01niTJPA/kKH0i7e0rLZBCwauQFcRE54=
```
Comment (somaritane) [inline go.sum:319]: Probably, slipped through?

Comment (kuritka) [reply go.sum:319]: Hi, unfortunately not :   
```sh
❯ go mod graph | grep zapr
sigs.k8s.io/controller-runtime@v0.6.4 github.com/go-logr/zapr@v0.1.0
sigs.k8s.io/controller-runtime@v0.4.0 github.com/go-logr/zapr@v0.1.0
```

graph command shows what is dependent on the left and dependencies on the right. if package has dependencies, it is shown in the table.

Comment (somaritane) [reply go.sum:319]: I see.

**Review by somaritane — APPROVED** [review]
lgtm

**Review by ytsarev — APPROVED** [review]

**Pull request merged**